pkg/kubelet/kubelet_pods.go

```diff
@@ -1649,13 +1649,16 @@ func (kl *Kubelet) GetPortForward(podName, podNamespace string, podUID types.UID
 }
 
 // cleanupOrphanedPodCgroups removes cgroups that should no longer exist.
-// it reconciles the cached state of cgroupPods with the specified list of runningPods
-func (kl *Kubelet) cleanupOrphanedPodCgroups(cgroupPods map[types.UID]cm.CgroupName, activePods []*v1.Pod) {
-	// Add all running pods to the set that we want to preserve
+// it reconciles the cached state of cgroupPods with the specified list of runningPods and activePods
+func (kl *Kubelet) cleanupOrphanedPodCgroups(cgroupPods map[types.UID]cm.CgroupName, activePods []*v1.Pod, runningPods []*kubecontainer.Pod) {
+	// Add all active and running pods to the set that we want to preserve
 	podSet := sets.NewString()
 	for _, pod := range activePods {
 		podSet.Insert(string(pod.UID))
 	}
+    for _, pod := range runningPods {
```

Comment:
this line has small indentation issue.
please use a tab here.

Reply:
done for latest commit